src/transformers/data/data_collator.py

```diff
@@ -328,7 +338,9 @@ def torch_call(self, features):
                 [self.label_pad_token_id] * (sequence_length - len(label)) + list(label) for label in labels
             ]
```

Comment:
If the `labels` are tensors, `list(label)` won't convert the `label` to a list, you need to call `tolist()`.